tsl/test/sql/continuous_aggs_union_view.sql

```diff
@@ -0,0 +1,119 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+-- disable background workers to make results reproducible
+\c :TEST_DBNAME :ROLE_SUPERUSER
+SELECT _timescaledb_internal.stop_background_workers();
+\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
```

Comment:
I suggest `SET ROLE :ROLE_DEFAULT_PERM_USER` to make this easier to run through a debugger.